src/main/java/com/iota/iri/conf/DbConfig.java

```diff
@@ -54,6 +54,34 @@
      */
     boolean isRescanDb();
 
+    /**
+     * Default Value: {@value BaseIotaConfig.Defaults#TX_BATCH_WRITE}
+     *
+     * @return {@value DbConfig.Descriptions#TX_BATCH_WRITE}
+     */
+    long getTxBatchWrite();
+
+    /**
+     * Default Value: {@value BaseIotaConfig.Defaults#MILESTONE_BATCH_WRITE}
+     *
+     * @return {@value DbConfig.Descriptions#MILESTONE_BATCH_WRITE}
+     */
+    int getMilestoneBatchWrite();
+
+    /**
+     * Default Value: {@value BaseIotaConfig.Defaults#TX_BATCH_EVICTION_COUNT}
+     *
+     * @return {@value DbConfig.Descriptions#TX_BATCH_EVICTION_COUNT}
+     */
+    int getTxBatchEvictionCount();
+
+    /**
+     * Default Value: {@value BaseIotaConfig.Defaults#MILESTONE_BATCH_EVICTION_COUNT}
+     *
+     * @return {@value DbConfig.Descriptions#MILESTONE_BATCH_EVICTION_COUNT}
+     */
+    int getMilestoneBatchEvictionCount();
```

Comment:
I didn't see you utilize `saveBatch` for optimizing evictions (since there are no changes to TVM and AVM I am sure you didn't)

So please optimize it

Reply:
This is done in the cache optimisation PR

Reply:
all this last minute stuff just gets me confused

Reply:
We will merge this PR then after cache-optimization